### ISSUE TITLE
Require Admin Access to see Library Manager Page

### ISSUE THREAD

**Issue by PlasmaIntec** [wwu-cs/Submitty#87]
Related to #86, we should eventually separate the Admin and Instructor roles as only the Admin should be allowed to access the Library Manager Page.

**Comment by sballance** [comment]
I'll go ahead and take this issue.

**Comment by sballance** [comment]
@PlasmaIntec @dakriy are we wanting to limit this to only superusers or should instructors be able to see it as well? I was originally just allowing superusers to access the post routes but I thought it would be a good idea to check before continuing. I don't want to allow instructors to access the page if they can't submit the post request.

**Comment by dakriy** [comment]
@sballance Yeah, just protect the whole controller with the same access level. I'd say only allow super user for now, and make it easy to change that access level, possibly in the homework_library.json config file, so that it is easily changeable.
